test/powershell/Language/Parser/Conversions.Tests.ps1

```diff
@@ -78,6 +78,10 @@ Describe 'conversion syntax' -Tags "CI" {
             $result -join ";" | Should -Be ($Elements -join ";")
         }
     }
+
+    It 'Should not convert invalid strings to type name' {
+        'int]whatever' -as [type] | Should -Be $null
+    }
```

Comment:
```suggestion
    }
    
    It 'Should not convert invalid strings to type name using left-hand side operator' {
        [Type] 'int]whatever' | Should -Throw
    }
```

Reply:
What is `-ErrorId` we expect?

Reply:
Probably`InvalidArgument` based on this experiment:

```
> [Type] 'int]whatever'

IsPublic IsSerial Name                                     BaseType
-------- -------- ----                                     --------
True     True     Int32                                    System.ValueType

@andys-mac-mini ~ 
> [Type] 'intwhatever' 
InvalidArgument: Cannot convert the "intwhatever" value of type "System.String" to type "System.Type".
```

Reply:
I mean we usually use `-ErrorId` in `Should -Throw`.

Reply:
Ah, well I don't know 🙃